subsys/net/ip/l2/ethernet.c

```diff
@@ -153,6 +156,9 @@ static enum net_verdict ethernet_recv(struct net_if *iface,
 		NET_DBG("ARP packet from %s received",
 			net_sprint_ll_addr((u8_t *)hdr->src.addr,
 					   sizeof(struct net_eth_addr)));
+#ifdef CONFIG_NET_IPV4_AUTO
+		net_ipv4_autoconf_input(iface, pkt);
```

Comment:
Shouln't we return the verdict to the caller here, so "return net_ipv4_autoconf_input(...)", now we always do arp thing regardless of autoconf return value?

Reply:
not sure, I think the pkt should in any case reach the arp handler in the next function call after this one, can you add more details?

Reply:
I am just asking this, the net_ipv4_autoconf_input() line just looked suspicious. If you are sure this is ok, then no worries here then.

Reply:
should be fine as is